docs/android/get-started/installation/openjdk.md

```diff
@@ -61,16 +60,18 @@ Point the IDE to the new JDK:
 
 ![Setting the JDK path for the Microsoft Mobile OpenJDK distribution on Windows](openjdk-images/vs.png)
 
-## Revert
+## Known Issues
+
+This is a new feature that affects all Android developers, and we are working hard to resolve issues as quickly as possible. Here are some of the known scenarios and how you can work through them to unblock yourself while we fix things on our end.
 
-To revert to the Oracle JDK, change the Java SDK location to the previously-used Oracle JDK path and rebuild the solution. On Mac, you can revert to the Oracle JDK path by clicking **Reset to Defaults**.
+### Package 'OpenJDKV1.RegKey,version=1.8.0.9,chip=x64' failed to install
 
-If you have any problems with the Microsoft Mobile OpenJDK distribution, please report issues by using the feedback tool in your IDE so that they can be tracked and corrected quickly.
+This may be an issue in some corporate environments. OpenJDK is already on the machine - follow the troubleshooting steps above to point your IDE to the correct location. You can follow the status of the issues [here](https://developercommunity.visualstudio.com/content/problem/382549/packageidopenjdkv1regkeypackageactioninstallreturn.html).
```

Comment:
"troubleshooting steps above" can be a link ie `[troubleshooting steps above](#troubleshooting)`